macos/ReactTestApp/AppDelegate.swift

```diff
@@ -300,13 +304,11 @@ extension AppDelegate {
         window.contentViewController?.view = rootView
 
         #if DEBUG
-        if Session.shouldRememberLastComponent {
-            rememberLastComponentMenuItem.state = .on
-        }
-
         showReactMenu()
         #endif // DEBUG
     }
+
+    private func onRememberLastComponentSelectedInternal(_: NSMenuItem) {}
```

Comment:
So singleAppMode doesn't need to care about this so it's an empty function? cool :P

Reply:
It doesn't make sense since you'll always launch into the app declared in `app.json`. But the Storyboard file needs this to be present ¯\_(ツ)_/¯

If we recreate whatever the Storyboard does programmatically, we can remove this empty function.

Reply:
Somewhat related: https://github.com/microsoft/react-native-test-app/issues/111

Reply:
Somewhat related, visionOS doesn't support storyboards (or was it xibs?) so I think so it would be good to remove uses in general